### ISSUE TITLE
ONE request should have ONE tab

### ISSUE THREAD

**Issue by Wimmmmm** [a85/POSTMan-Chrome-Extension#5573]
My request is very simple: ONE request should have ONE tab. 
Do not allow multiple tabs per request.

I did some fieldwork before posting this request:

- I read many other request related to tabs
- I also read the general dev team desire to "save as much work" as possible, therefore doing tricks when deleting a request

but I think that in terms of UX, you made one fundamental decision, which is not helping us users the way you intended: you allow a single request to be openen in multiple tabs.

This causes many unusual side-effects - and personally I find them more annoying than helpful:
- multiple tabs of the same object (a request)
- you can have edits in more than one tab - but they don't merge, they just overwrite one each other
- after deleting a request, you might still have tabs open, and then you need to decide what to do with them. As discussed elsewhere already: a delete is a delete. When I decide to remove something, your application should remove it everywhere. Leaving tabs with [Delete] still leaves work for me to doublecheck my own actions & I don't like that extra effort. Maybe I deleted this request 3 days ago.
- each time you open a request, a new tab appears. I don't want this, I want to see the request in the single tab it is already open. I can't search the tabs, and with 20+ tabs open, I need some navigation system to find my way. Oh, we HAVE a navigation system for this: the left-hand bar!!
- Basically I learned myself, when closing all tabs, to hit the "Ignore all changes and close anyway" button. So your double-check feature is not producing the desired outcome...

It's more code for you to maintain, and it's not giving me a desired UX experience.
Therefore my request: ONE request should have ONE tab.

Good luck!

**Comment by vkaegis** [comment]
@Wimmmmm Thanks for writing in! As you could find with your fieldwork, we have a few issues around tabs that we're considering for improvements, possibly opening up configurations around this. Still exploring ideas, so no ETA on this just yet. Will keep you posted here.

**Comment by a85** [comment]
Editing a request is not the only reason people open tabs. While debugging or testing, people open multiple tabs and get different responses or even do different variations of the same request. In a single tab scenario this wouldn't be possible. As @vkaegis mentioned, we are improving the tab workflow regularly and will keep this issue open till then.
